### PR TITLE
Updates - Consul version, VPC CloudTemplate and ACM CloudTemplate

### PR DESCRIPTION
**Pull request by gautambaghel**
Updated the Consul version, updated the VPC with a newer template, and the ACM cert generation CloudTemplate

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/quickstart-hashicorp-consul.template [hunk 1]
```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: 2010-09-09
-Description: "HashiCorp Consul (Please do not remove) Aug,28,2019 (qs-1qup6ra8j)"
+Description: "HashiCorp Consul (Please do not remove) Aug,28,2019 QS(0036)"
```
Comment (wellsiau-aws) [inline templates/quickstart-hashicorp-consul.template:2]: keep the QuickStart code (`qs-1qup6ra8j`) in the description

**Review by wellsiau-aws — COMMENTED** [review]
since you are using [quickstart-aws-acm-certificate](https://github.com/aws-quickstart/quickstart-aws-acm-certificate), you can remove the following files:

- [acm-cert-r53-record.template.yaml](https://github.com/gautambaghel/quickstart-hashicorp-consul/blob/main/templates/acm-cert-r53-record.template.yaml)
- [copy-lambdas.template](https://github.com/gautambaghel/quickstart-hashicorp-consul/blob/main/templates/copy-lambdas.template)

File: templates/quickstart-hashicorp-consul-master.template [hunk 2]
```diff
@@ -267,22 +295,40 @@ Conditions:
 #         AssertDescription: "KeyPairName must be provided." 
 Resources:
   VPCStack:
-    Type: 'AWS::CloudFormation::Stack'
+    Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+          - W9198
```
Comment (wellsiau-aws) [inline templates/quickstart-hashicorp-consul-master.template:302]: add short explaination why we ignoring this check

File: templates/quickstart-hashicorp-consul-master.template.yaml [hunk 3]
```diff
@@ -267,23 +295,40 @@ Conditions:
 #         AssertDescription: "KeyPairName must be provided." 
 Resources:
   VPCStack:
-    Type: 'AWS::CloudFormation::Stack'
+    Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+          - W9198
```
Comment (wellsiau-aws) [inline templates/quickstart-hashicorp-consul-master.template.yaml:302]: add short explanation why we skip this check

**Comment by wellsiau-aws** [comment]
this will close issue #83

**Comment by wellsiau-aws** [comment]
overall looks good, see my previous comment about tidying up unused files.
I believe there is two additional parameters that user need to supply and we need to update the documentation as well.

**Review by wellsiau-aws — DISMISSED** [review]
LGTM

**Review by gargana — DISMISSED** [review]
LGTM

**Comment by gargana** [comment]
CI Failed:
```
[INFO   ] : consul us-west-2 CREATE_IN_PROGRESS
[INFO   ] : consul us-east-1 CREATE_IN_PROGRESS
[INFO   ] : consul us-east-2 CREATE_IN_PROGRESS
[INFO   ] : consul ap-south-1 CREATE_IN_PROGRESS
[INFO   ] : consul eu-west-1 CREATE_IN_PROGRESS
[INFO   ] : consul eu-north-1 CREATE_IN_PROGRESS
[INFO   ] : consul eu-central-1 CREATE_IN_PROGRESS
[INFO   ] : consul eu-west-3 CREATE_IN_PROGRESS
[INFO   ] : consul eu-west-2 CREATE_IN_PROGRESS
[INFO   ] : consul ap-southeast-1 CREATE_IN_PROGRESS
[INFO   ] : consul ap-northeast-1 CREATE_IN_PROGRESS
[INFO   ] : consul ap-southeast-2 CREATE_IN_PROGRESS
[INFO   ] : consul ap-northeast-2 CREATE_IN_PROGRESS
[INFO   ] : consul sa-east-1 CREATE_IN_PROGRESS
[ERROR  ] : consul us-east-2 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 1 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul us-east-1 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 3 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul ap-south-1 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 1 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul eu-west-1 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 2 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul eu-central-1 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 1 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul eu-west-3 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 3 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul eu-west-2 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 2 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul ap-southeast-1 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 2 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul ap-northeast-1 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 3 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul ap-southeast-2 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 2 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul ap-northeast-2 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 1 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul sa-east-1 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 1 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul us-west-2 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 1 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html
[ERROR  ] : consul eu-north-1 CREATE_FAILED
[ERROR  ] :     ConsulClientAsg Received 3 FAILURE signal(s) out of 3.  Unable to satisfy 100% MinSuccessfulInstancesPercent requirement
[ERROR  ] :     ACMCertificate S3 error: Access Denied
```
judging from the errors the ACMCertificate file is missing or the path is incorrect.

**Comment by gautambaghel** [comment]
Yup, fixing it soon

**Comment by wellsiau-aws** [comment]
local taskcat test on my end confirmed that the ACMCertificate access denied was resolved, requesting another e2e @gargana

File: templates/quickstart-hashicorp-consul.template (hunk 1 above)
Comment (gargana) [reply templates/quickstart-hashicorp-consul.template:2]: This doesn't look like it is reverted. Please keep the (qs-1qup6ra8j) in the description.

**Review by gargana — CHANGES_REQUESTED** [review]
Just one issue that I can see. Once that change is in we should be good to ship this.

**Comment by gautambaghel** [comment]
Not sure why it changed again, I thought I fixed it

**Review by gargana — APPROVED** [review]
LGTM

**Pull request merged**